core/templates/dev/head/forms/formBuilder.js

```diff
@@ -672,7 +672,13 @@ oppia.factory('rteHelperService', [
           elt.find(
             'img.oppia-noninteractive-' + componentDefn.name
           ).replaceWith(function() {
-            var jQueryElt = $('<' + this.className.split(' ')[0] + '/>');
+            // Look for a class name starting with oppia-noninteractive-*.
+            var tagNameMatch = /(^|\s)(oppia-noninteractive-\w+)/.exec(
```

Comment:
Oops missed that out, thanks!. Hmm just wondering, is there a possibility that uppercase characters or digits might be used too?